src/one_dragon/base/operation/operation.py

```
@@ -891,6 +891,27 @@ def round_by_ocr(
         else:
             return self.round_retry(f'找不到 {target_cn}', wait=retry_wait, wait_round_time=retry_wait_round)
 
+    def ocr(self, screen: MatLike, screen_name: str, area_name: str,
+                           success_wait: Optional[float] = None, success_wait_round: Optional[float] = None,
```

Ruff (0.13.1)

Unused method argument: `success_wait`

(ARG002)